documentation/source/NuRadioMC/pages/HDF5_structure.rst

```diff
@@ -32,7 +47,8 @@ The HDF5 files can be thought of as a structured dictionary:
 - Finally, the ``station_<station_id>`` key contains slightly more detailed information (triggers, propagation times, amplitudes...) at the level of individual channels :ref:`for each station <NuRadioMC/pages/HDF5_structure:Station data>`.
 
 HDF5 file attributes
-____________________
+----------------------
+
```

Comment:
The line style controls the level of the heading - I would have left `f.attrs` as a subheading of the HDF5 structure heading, at the same level as `f[key]` and `f[station]`, but I'm not going to argue if you disagree.

Reply:
I am happy to change that back. I think I just wanted to not throw an error.